demo/nbody/src/main.rs

```diff
@@ -1,111 +1,123 @@
+ #![feature(augmented_assignments)]
```

Comment:
Hmm, I'm a bit disappointed to take this off of stable. But it's just a demo I guess.